test/integration/include_custom_model.sdf

```diff
@@ -98,8 +98,43 @@
               <plugin name="top_plugin3"/>
             </visual>
 
+            <collision name="top_collision" action="replace">
+              <geometry>
+                <sphere>
+                  <radius>0.65</radius>
+                </sphere>
+              </geometry>
+            </collision>
+
           </link>
 
+          <!-- attribute test get's stripped during replace -->
+          <collision element_id="right_wheel::collision" test="hello" name="right_wheel_collision" action="replace">
```

Comment:
why does this one get stripped?

Reply:
Unknown attributes gets "stripped" (ignored) by `readXml` here: 
https://github.com/osrf/sdformat/blob/bf1c809f20eec80003f7525641410c590cf5760c/src/parser.cc#L893-L898

I left it that way since this is how SDF currently works. I spoke to @azeey & this is most likely an issue that needs to be addressed in the future (but not part of param passing).

Reply:
got it, thanks.